src/fosslight_binary/_jar_analysis.py

```
@@ -3,309 +3,405 @@
-            # Get OSS Version
-            oss_version = pkg_info.get('id').split('@')[1]
-    except Exception as ex:
-        logger.debug(f"Error to get value for oss name and version: {ex}")
-    return oss_name, oss_version
+    if skip_central:
+        central_info = {}
+        timed_out = False
+    else:
+        central_info, timed_out = _search_central_by_sha1(sha1, timeout=search_timeout)
+        if timed_out:
+            logger.debug(f"{rel_path}: Central SHA-1 search timed out – will retry")
```

Ruff (0.15.13)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)